app/admin.py

```diff
@@ -108,6 +109,8 @@ def post(self):
                         'deactivation_message_html',
                         'keywords',
                         'launch_status',
+                        'admin_user_email',
```

Comment:
Do you intend these configs to be global (not configured per repository)? If so, you should move these to "elif self.params.operation == 'save_global'" block below. Also, you should add these to "Global settings" section in admin.html.template instead.